### PR TITLE
Fix parent path handling when importing test projects

### PR DESCRIPTION
**Pull request by tsmaeder**
### What does this PR do?
This is a fixup for breaking test project import via zipping directories. Streamlined adding files to the zip a bit in the process.

### What issues does this PR fix or reference?
#10764

### CODE REVIEW

File: core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java [hunk 1]
```diff
@@ -38,91 +37,88 @@
...
+                } catch (IOException e) {
+                  throw new ExceptionWrapper(e);
+                }
+              });
+    } catch (ExceptionWrapper e) {
+      throw (IOException) e.getCause();
     }
   }
 
+  private static String relativePath(Path root, Path f) {
```
Comment (tolusha) [inline core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java:120]: AFAIK these changes won't be approved for the master branch.

Comment (tolusha) [reply core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java:120]: They just don't accept refactoring in those components.

Comment (tolusha) [reply core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java:120]: @skabashnyuk Am I right?

File: core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java [hunk 2]
```diff
@@ -1,27 +1,26 @@
 /*
- * Copyright (c) 2012-2018 Red Hat, Inc.
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * Copyright (c) 2012-2018 Red Hat, Inc. This program and the accompanying materials are made
```
Comment (skabashnyuk) [inline core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java]: I believe this changes in the license are made by mistake.

File: core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java (hunk 1 above)
Comment (skabashnyuk) [reply core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java:120]: I'm ok if unit and selenium test will be ok.

**Review by skabashnyuk — APPROVED** [review]
In general, I'm ok with this PR. Maybe it can be ported in master first?

**Review by tolusha — APPROVED** [review]

**Review by svor — APPROVED** [review]

File: core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java (hunk 2 above)
Comment (tsmaeder) [reply core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/ZipUtils.java]: Hmh...I might have had "skip license check" flag on when building this branch. Good catch, I'll fix it.

**Pull request merged**